docs/develop/dotnet/versioning.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: versioning
+title: Versioning .NET SDK feature guide
+sidebar_label: Versioning
+description: The Versioning section of the Temporal Developer's guide covers how to update Workflow Definitions without causing non-deterministic behavior in current long-running Workflows.
+toc_max_heading_level: 4
+keywords:
+  - best practices
+  - code sample
+  - deployment
+  - deployment safety
+  - deprecated patches
+  - how-to
+  - patching
+  - dotnet
+  - dotnet sdk
+  - version
+  - versioning
+  - workflow completion
+  - workflow history
+  - workflow transition
+tags:
+  - best-practices
+  - code-sample
+  - deployment
+  - deployment-safety
+  - deprecated-patches
+  - how-to
+  - patching
+  - dotnet
+  - dotnet-sdk
+  - version
+  - versioning
+  - workflow-completion
+  - workflow-history
+  - workflow-transition
+---
+
+## Introduction to Versioning
+
+## How to use the .NET SDK Patching API {#.NET-sdk-patching-api}
+
+<!-- CONTENT-->
+
+**Steps involved in implementing patching using the Temporal .NET SDK**
+
+<!-- CONTENT AND CODE EXAMPLE-->
+
+### Patching in new code {#using-patched-for-workflow-history-markers}
+
+<!-- CONTENT AND CODE EXAMPLE-->
+
+### Understanding deprecated Patches in the .NET SDK {#deprecated-patches}
+
+<!-- CONTENT-->
+
+### Safe Deployment of post_patch_activity {#deploy-new-code}
+
+<!-- CONTENT-->
+
+## How to use Worker Versioning in .NET {#worker-versioning}
+
+<!-- CONTENT-->
+
+### Assign a Build ID to your Worker
+
+<!-- CONTENT-->
+
+### Specify versions for Commands
+
+**How to specify versioning intent**
+
+<!-- CONTENT-->
+
+**How to use the latest default version of an Activity**
+
+<!-- CONTENT-->
```

Comment:
I don't believe these sections should be added at this time knowing they are already obsolete pending new versioning implementation